be/src/exec/spill/serde.h

```diff
@@ -41,8 +44,8 @@ class Serde {
     virtual Status serialize(SerdeContext& ctx, const ChunkPtr& chunk, BlockPtr block) = 0;
     // deserialize data from block, return the chunk after deserialized
     virtual StatusOr<ChunkUniquePtr> deserialize(SerdeContext& ctx, const BlockPtr block) = 0;
+
+    static StatusOr<SerdePtr> create_serde(const ChunkBuilder& chunk_builder, const CompressionTypePB& compress_type);
```

Comment:
just using create as function name is enough, since we have move it into Serde class

Reply:
to match our code-style